knowledge_base/VERSIONING_ISSUES.md

```diff
@@ -0,0 +1,23 @@
+# Versioning Issues
+
+## Pytorch version
+This issue is related to https://github.com/pytorch/rl/issues/689. Using PyTorch versions lower < 1.13 and installing stable package leads to the following error:
+```
+ImportError: /usr/local/lib/python3.7/dist-packages/torchrl/_torchrl.so: undefined symbol: _ZN8pybind116detail11type_casterIN2at6TensorEvE4loadENS_6handleEb
+```
+This is probably due to some incompatibility (tensor casting?) of torch C++ bindings between version 1.12 and 1.13. 
+
+
+### How to reproduce
+1. Create an Colab Notebook (at 24/11/2022 Colab enviroment has Python 3.7 and Pytorch 1.12 installed by default).
+2. ``` !pip install torchrl ```
+3. ``` import torchrl ```
+
+### Workarounds
+1. Install torch 1.13 or above.
+2. Install latest version of functorch. This version depends on torch 1.13 and will install it for you. 
```

Comment:
Here I would be a bit more precise:
If you install torch >= 1.13, functorch will be part of the package
Otherwise (for torch <=1.12.1) you should install functorch via `pip install functorch==0.2.0`

If users choose to select (2) they will need to install from source anyway (as specified in (3))

So i think we should say:

> There are two workarounds to this issue:
> 1. Install/upgrade to the latest pytorch
> 2. Install functorch and install the library from source